python/ray/data/_internal/logical/operators/map_operator.py

```diff
@@ -18,10 +18,33 @@
 
 
 class AbstractMap(LogicalOperator):
-    """Abstract class for logical operators should be converted to physical
+    """Abstract class for logical operators that should be converted to physical
```

Comment:
btw, this comment is becoming correct from now.

Reply:
sorry what do you mean? Previously all `AbstractMap` operator is also converted to `MapOperator` in `plan_map_op` right? It's just `Read` is converted to `MapOperator(InputDataBuffer)`

Reply:
yeah I mean the Read is not captured with AbstractMap although it's converted to MapOperator as well. Now we unify all of them which is a bit more cleaner.

Reply:
Ok got it.